packages/core/src/components/forms/_input-group.scss

```diff
@@ -78,20 +92,21 @@ $input-button-height-large: $pt-button-height !default;
     padding-bottom: 0;
   }
 
-  .pt-icon {
-    @include pt-icon($pt-icon-size-standard);
-
-    // bump icon up so it sits above input
-    z-index: 1;
+  > .pt-icon-standard {
     margin: 0 ($pt-input-height - $pt-icon-size-standard) / 2;
-    line-height: $pt-input-height;
+  }
+
+  .pt-icon-standard {
```

Comment:
we used `.pt-icon` before so that we could adapt the icon size if `large`. seems we never actually use large icons, so this is probably fine.

Reply:
@giladgray stashing this and punting for now